features/storage/csi_resize.feature

```diff
@@ -75,3 +75,86 @@ Feature: CSI Resizing related feature
     Examples:
       | sc_name |
       | gp2-csi | # @case_id OCP-25809
+
+  # @author ropatil@redhat.com
+  @admin
+  Scenario Outline: [CSI Resize] offline volume expansion from 1Gi to 2Gi
+    Given the master version >= "4.8"
+    Given I have a project
+ 
+    # Create pvc with csi storage class 
+    Given I obtain test data file "storage/misc/pvc.json"
+    When I create a dynamic pvc from "pvc.json" replacing paths:
+      | ["metadata"]["name"]                         | mypvc         |
+      | ["spec"]["storageClassName"]                 | <sc_name>     |
+      | ["spec"]["accessModes"][0]                   | ReadWriteOnce |
```

Comment:
Let's remove this line.

Reply:
It has no side impact, but if we use the default value and there is no specific meaning or different from others, no need to list it.
For another storage size, although 1G is the default value, it will help up to understand the resize from 1G to 2G, so I think it is meaningful here.